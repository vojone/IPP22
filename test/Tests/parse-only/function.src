.Ippcode22


CALL abc


LABEL   abc
PUSHFRAME
WRITE   string@Hello
POPFRAME
RETURN
