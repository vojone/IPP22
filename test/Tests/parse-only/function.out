<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode22">
    <instruction opcode='CALL' order='1'>
        <arg1 type='label'>abc</arg1>
    </instruction>
    <instruction opcode='LABEL' order='2'>
        <arg1 type='label'>abc</arg1>
    </instruction>
    <instruction opcode='PUSHFRAME' order='3'>
    </instruction>
    <instruction opcode='WRITE' order='4'>
        <arg1 type='string'>Hello</arg1>
    </instruction>
    <instruction opcode='POPFRAME' order='5'>
    </instruction>
    <instruction opcode='RETURN' order='6'>
    </instruction>
</program>
